application/templates/cronjob.yaml

```diff
@@ -31,10 +31,17 @@ spec:
 {{ end }}
   jobTemplate:
     spec:
+{{- if $job.activeDeadlineSeconds }}
```

Comment:
You can use with in here

Reply:
Changed in [49d195c](https://github.com/stakater/application/pull/265/commits/49d195c7bc40c21c578d5fc03c9445fa6a517d80)